### ISSUE TITLE
Option to keep the last position we were in a folder placed in bookmark toolbar.

### ISSUE THREAD

**Issue by rpgmaniac** [brave/brave-browser#13558]
When we save a new bookmark in a folder that we have placed in the bookmark toolbar the newest site is going to the bottom of the folder, the folder in Brave doesn't keep the last position & always reset at the top & this is seriously very annoying & as it force me to always w8 until it goes all the way to the bottom from the top when I wanna check a bookmark that I have saved recently, in firefox btw this is not how this worked it always kept the last position, plz change this & make it by default to keep the last position.

**Comment by rebron** [comment]
@rpgmaniac Seems like you're looking for a similar experience from Firefox but we need much more clarification and clearer steps on what you're looking for. 

When you add a bookmark to Brave in a folder via the bookmark button near the url bar, that bookmark goes to the bottom. You want bookmarks saved to the top is your feature request?

You could also just drag and drop the url into the folder and when it opens, place it at the top or wherever you'd like within the folder.

**Comment by rpgmaniac** [comment]
@rebron 

> You want bookmarks saved to the top is your feature request?

Nop this is not what I'm asking, I'm asking to remove the auto-reset-at-the-start from folders every time we take the mouse out of a folder, I want the folder to remember (save) the last position (link) I had my mouse before I close the folder & when I return to the folder to be at the exact same spot & not reset to the top at the first link, this is extremely frustrating & annoying & this didn't happen in firefox.

**Comment by rpgmaniac** [comment]
@rebron So I saw that you set this as priority 5 which is very-very disappointing as it's not something crazy to do & I was hoping that those features which doesn't require much work would be placed ahead in line in order to make the browser more functional & offer a better experience to new users, this is breaking for me as I use multiple folders to save my bookmarks & it's annoying as hell to navigate through the hundreds of bookmarks I have in those folders when the placement reset every time to the top, plz can you fix this? I'm not asking something that require months of coding how hard can this be?

**Comment by tmikaeld** [comment]
I'm having the same issue:
- Dropping a new bookmark into a folder in the bookmark toolbar, should not close the entire menu.
- Moving bookmarks or folders around, closes the entire menu.
- Deleting bookmarks, closes the entire menu.

This makes it very hard to use the bookmarks toolbar at all, Chrome have the same issue, Firefox doesn't.

**Comment by rpgmaniac** [comment]
> I'm having the same issue:
> 
> * Dropping a new bookmark into a folder in the bookmark toolbar, closes the entire menu.
> * Moving bookmarks or folders around, closes the entire menu.
> * Deleting bookmarks, closes the entire menu.
> 
> This makes it very hard to use the bookmarks toolbar at all, Chrome have the same issue, Firefox doesn't.

As you can see this thread created 8 months ago, 8 F months ago, when I change from FF to Brave there was so many issues I had with this browser I did a few other threads also the same period I did this one, guess what happen after all this time, not even 1 change, nothing, so based on what I have seen the devs either don't give a F or they are so slow that you might need to w8 a lifetime to actually see changes happen, so I suggest you to not expect anything else you will be greatly disappointed.

**Comment by tmikaeld** [comment]
@rpgmaniac Yeah, I'm used to no action, I tried creating the same on Chromium but it was just deleted immediately. Guess they just don't care about bookmarks.
